### PR TITLE
ZipShip Hackathon Updates

### PR DESCRIPTION
**Pull request by brady-spiva**
- updated the UBI base image to a more recent version which includes Molecule v6
- changed the collection's directory structure and naming conventions. This involved reorganizing ALL of the files
- updated the devfile to include environment variable: ANSIBLE_COLLECTIONS_PATH
- add a `requirements.yml` file, and update the molecule stages to include `dependency`
- updated the README for the demo

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # quay.io/ansible/creator-ee:v0.20.0
-FROM quay.io/ansible/creator-ee@sha256:72acf1476d32a7e56ec9045d8147eee010d8eecfc52590a3b8a7a620985696d4
+FROM quay.io/ansible/creator-ee@sha256:cffd700f8199306cec8d2d0eae202280d3fbf5e3f68c860b492de2eb511c25c0
```
Comment (svor) [inline Dockerfile]: It seems that the image with this digest doesn't exist.
I see that [quay.io/ansible/creator-ee@sha256:bf95ae745601a9d199eeb12807be24413fc2d6488016069816ee74342c2c255b](https://quay.io/repository/ansible/creator-ee?tab=tags&tag=latest) is the latest available.
If it is a plan to use it, could you please update the commented line above with appropriate tag.

File: collections/ansible_collections/sample_namespace/sample_collection/extensions/molecule/default/requirements.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+collections:
+  - 'community.okd'
```
Comment (svor) [inline collections/ansible_collections/sample_namespace/sample_collection/extensions/molecule/default/requirements.yml]:
```suggestion
  - 'community.okd'

```

File: Dockerfile (hunk 1 above)
Comment (svor) [reply Dockerfile]: cc @brady-spiva

Comment (brady-spiva) [reply Dockerfile]: @svor thank you for catching this, at the time we created this PR the image with that digest was there 🤷

**Comment by brady-spiva** [comment]
@svor @brandor5 I resolved a merge conflict, and this is good to go now!

**Pull request merged**